lib/crowbar/client/command/proposal.rb

```diff
@@ -22,10 +24,16 @@ module Proposal
 
         included do
           desc "Proposal specific commands"
-          command :proposal do |parent|
+          command "proposal" do |parent|
             parent.desc "Show a list of available proposals"
             parent.arg :barclamp
-            parent.command :list do |c|
+            parent.command "list" do |c|
+              c.desc "Format of the resulting output"
```

Comment:
Makes sense to print it somewhere instead of only in error case with invalid format (Invalid format, valid formats: table, json, plain)